src/diffusers/pipelines/audio_diffusion/mel.py

```diff
@@ -13,17 +13,12 @@
 # limitations under the License.
 
 
-import warnings
+import numpy as np  # noqa: E402
 
 from ...configuration_utils import ConfigMixin, register_to_config
 from ...schedulers.scheduling_utils import SchedulerMixin
 
 
-warnings.filterwarnings("ignore")
```

Comment:
This warnings filter silences all warnings. Need to remove to see the attention block deprectation warning